core/src/main/java/com/linecorp/armeria/common/stream/FilteredStreamMessage.java

```diff
@@ -103,6 +103,8 @@ protected Throwable beforeError(Subscriber<? super U> subscriber, Throwable caus
         return cause;
     }
 
+    // TODO(ikhoon): Add onCancel() handler
```

Comment:
Oh, I added it so we can remove this line. 😄

Reply:
Thank!